Swift/Collection.swift

```diff
@@ -302,6 +303,42 @@ public final class Collection : CollectionChangeObservable, Indexable, Equatable
         return ListenerToken(token)
     }
     
+    // MARK: Combine Publisher
+    
+    @available(iOS 13.0, *)
+    public func changePublisher(on queue: DispatchQueue = .main) -> AnyPublisher<CollectionChange, Never> {
+        let subject = PassthroughSubject<CollectionChange, Never>()
+        var token: ListenerToken?
+        
+        token = self.addChangeListener(queue: queue) { change in
```

Comment:
There is no need to have the separate lines for defining the token variable.

```
let token = self.addChangeListener(queue: queue) { .... }
```

Reply:
you are right, `addChangeListener` doesn't return Optional.

Reply:
No need to have a separate line here also for the optional case.